editor/src/core/model/element-metadata-utils.ts

```diff
@@ -1449,3 +1421,9 @@ export function getSimpleAttributeAtPath(
     propsOrAttributes,
   )
 }
+function isGivenUtopiaElementFromMetadata(
```

Comment:
This is why the tests break.

I had no recollection of ever writing this function (instead of importing the real function I wrote in another file)

Turns out I clicked on the yellow bulb from muscle memory, and instead of importing the function, it HELPFULLY DECLARED IT FOR ME WTF

<img width="634" alt="image" src="https://user-images.githubusercontent.com/2226774/118159742-b3e34780-b41d-11eb-974d-2622cc0f1af0.png">